shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -381,6 +383,68 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
     sendViewportMetricsToFlutter();
   }
 
+  // TODO(garyq): Add support for notch cutout API
+  // Decide if we want to zero the padding of the sides. When in Landscape orientation,
+  // android may decide to place the software navigation bars on the side. When the nav
+  // bar is hidden, the reported insets should be removed to prevent extra useless space
+  // on the sides.
+  enum ZeroSides {
+    NONE,
+    LEFT,
+    RIGHT,
+    BOTH
+  }
```

Comment:
If this isn't mirroring an Android API, consider just having `LEFT` and `RIGHT` values, then using `EnumSet.allOf` and `EnumSet.noneOf` in `calculateShouldZeroSides` below.